### DISCUSSION TITLE
How to add `--force` to `npm` operations on a migration?

### DISCUSSION THREAD

**Discussion by Sorgrum** [storybookjs/storybook#26629] [Help]
### Summary

My project has an issue with a dependency, which requires all `npm` operations to use `--force`.

The following migration is failing because it can't install the needed dependency. 

I've tried installing the dependency manually before the migration is run, but it doesn't seem to help.

```
➜  npx storybook@latest migrate mdx-to-csf --glob './**/*.stories.mdx'                                                                                  

The CJS build of Vite's Node API is deprecated. See https://vitejs.dev/guide/troubleshooting.html#vite-cjs-node-api-deprecated for more details.
✅ Adding "@storybook/blocks" package

An error occurred while installing dependencies:
NPM error ERESOLVE - Dependency resolution error.

Please check the logfile generated at ./storybook.log for troubleshooting and try again.
Error: Error: NPM error ERESOLVE - Dependency resolution error.
```

**Comment by valentinpalkovic** [comment]
Hi @Sorgrum 

this is an interesting issue! I am curious about the following: 

Since we are just triggering the npm CLI, it should in theory just respect a local .npmrc file and its settings. Couldn't you temporarily just add the `force: true` setting to your local .npmrc and then trigger the automigration?
